test/functional/options/fcnotify_spec.lua

```diff
@@ -0,0 +1,313 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
related, why was `test/functional/autoread/` created? Don't need a subdirectory for every feature.